packages/suite-desktop/src-electron/electron.ts

```diff
@@ -118,8 +120,13 @@ const init = async () => {
         }
     };
 
-    mainWindow.webContents.on('will-navigate', handleExternalLink);
-    mainWindow.webContents.on('new-window', handleExternalLink);
+    mainWindow.webContents.on('new-window', (event, url, frameName) => {
```

Comment:
i'm not sure about removing 
`mainWindow.webContents.on('will-navigate', handleExternalLink);` if it not brake anything

Reply:
We have returned `mainWindow.webContents.on('will-navigate', handleExternalLink);` as it was removed by mistake. Decide what to do with it separately.